modules/theme-tailwind/organisms/CheckoutLoginModal.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <Modal @close="emit('close')">
+    <SfxForm class="space-y-6" @submit="onSubmit">
+      <FormInput name="password" type="password" :label="t('Password')" validators="required" />
+
+      <Button type="submit" color="primary" :loading="isLoading" class="w-full" data-cy="sign-in">
+        {{ t('Sign in') }}
+      </Button>
+    </SfxForm>
+  </Modal>
+</template>
+
+<script setup lang="ts">
+import useI18n from '#ioc/composables/useI18n'
+import SfxForm from '#ioc/components/SfxForm'
+import Modal from '#ioc/atoms/Modal'
+import Button from '#ioc/atoms/Button'
+import FormInput from '#ioc/molecules/FormInput'
+import useLoginCustomer from '#ioc/services/useLoginCustomer'
+import useCustomerStore from '#ioc/stores/useCustomerStore'
+import useGetCustomer from '#ioc/services/useGetCustomer'
+import useShowErrorNotification from '#ioc/composables/useShowErrorNotification'
+import { ref } from 'vue'
+
+const showErrorNotification = useShowErrorNotification()
+const loginCustomer = useLoginCustomer()
+const customerStore = useCustomerStore()
+const getCustomer = useGetCustomer()
+const { t } = useI18n()
+const isLoading = ref(false)
+const emit = defineEmits(['close'])
+
+const props = defineProps({
+  email: {
+    type: String,
+    required: true,
+  },
+})
+
+const onSubmit = async ({ password }: { password: string }) => {
+  try {
+    isLoading.value = true
+    await loginCustomer(props.email, password, { redirect: false })
```

Comment:
This is framework logic that should be encapsulated in some service (loginCustomerInCheckout). Why should programmer that wants to login customer in checkout know they shoudl disable the redirect and patch the store?